client/asset/eth/eth.go

```diff
@@ -121,6 +121,17 @@ var (
 			Repeatable: providerDelimiter,
 			Required:   true,
 		},
+		// Allow for single Provider setup, still nag the user to specify 2nd one for
+		// additional safety in case 1st provider fails during trade settlement.
+		{
+			Key:         providersKey,
+			DisplayName: "Provider",
+			Description: "Specify one or more providers. For infrastructure " +
+				"providers, use an https address. Only url-based authentication " +
```

Comment:
"use an https address"?  We're supposed to be recommending wss

Reply:
Changed this text a bit.